static/app/views/organizationGroupDetails/header.tsx

```diff
@@ -150,6 +151,14 @@ class GroupHeader extends React.Component<Props, State> {
         <div className={className}>
           <div className="row">
             <div className="col-sm-7">
+              <StyledBreadcrumbs
+                crumbs={[
+                  {label: 'Issues', to: `/organizations/${orgId}/issues/`},
```

Comment:
I think we need to include the `?project` query param here to ensure that navigating using the breadcrumbs will show the issue stream relevant to the issue detail's project.